kolibri/plugins/coach/class_summary_api.py

```diff
@@ -152,9 +155,27 @@ def get_num_correct(self, exam_log):
             .get("correct__sum")
         )
 
+    def get_num_answered(self, exam_log):
+        return (
+            exam_log.attemptlogs.values_list("item")
```

Comment:
Still need to make the same update here as was made above:
`examlog.attemptlogs.values_list("item", "content_id")`

Reply:
@rtibbles Should the change also apply to the `get_num_correct` method just above that `get_num_answered` method as well?

Reply:
Ah, yes - otherwise we might undercount.